integ-test/src/test/java/org/opensearch/sql/calcite/standalone/CalcitePPLEventstatsIT.java

```diff
@@ -0,0 +1,369 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.calcite.standalone;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_STATE_COUNTRY;
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_STATE_COUNTRY_WITH_NULL;
+import static org.opensearch.sql.util.MatcherUtils.rows;
+import static org.opensearch.sql.util.MatcherUtils.schema;
+import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
+import static org.opensearch.sql.util.MatcherUtils.verifyErrorMessageContains;
+import static org.opensearch.sql.util.MatcherUtils.verifySchema;
+
+import java.io.IOException;
+import java.util.List;
+import org.json.JSONObject;
+import org.junit.Ignore;
+import org.junit.jupiter.api.Test;
+import org.opensearch.client.Request;
+import org.opensearch.sql.legacy.TestsConstants;
+
+public class CalcitePPLEventstatsIT extends CalcitePPLIntegTestCase {
+  @Override
+  public void init() throws IOException {
+    super.init();
+    loadIndex(Index.STATE_COUNTRY);
+    loadIndex(Index.STATE_COUNTRY_WITH_NULL);
+  }
+
+  @Test
+  public void testEventstat() {
+    JSONObject actual =
+        executeQuery(
+            String.format(
+                "source=%s | eventstats count() as cnt, avg(age) as avg, min(age) as min, max(age)"
+                    + " as max",
+                TEST_INDEX_STATE_COUNTRY));
+
+    verifySchema(
+        actual,
+        schema("name", "string"),
+        schema("age", "integer"),
+        schema("state", "string"),
+        schema("country", "string"),
+        schema("month", "integer"),
+        schema("year", "integer"),
+        schema("cnt", "long"),
+        schema("avg", "double"),
+        schema("min", "integer"),
+        schema("max", "integer"));
+
+    verifyDataRows(
+        actual,
+        rows("John", "Canada", "Ontario", 4, 2023, 25, 4, 36.25, 20, 70),
+        rows("Jake", "USA", "California", 4, 2023, 70, 4, 36.25, 20, 70),
+        rows("Jane", "Canada", "Quebec", 4, 2023, 20, 4, 36.25, 20, 70),
+        rows("Hello", "USA", "New York", 4, 2023, 30, 4, 36.25, 20, 70));
```

Comment:
in scheam, 2nd column is age, but in result, 2nd column is country?

Reply:
The data order is the actually schema order. Let me change to `verifySchemaInOrder`